migrator/zenodo_rdm_migrator/transform/entries/records/records.py

```diff
@@ -26,11 +26,17 @@ def _id(self, entry):
 
     def _created(self, entry):
         """Returns the creation date of the record."""
-        return entry["created"]
+        value = entry["created"]
```

Comment:
It was very complex to create subclasses of entries and overwrite `Transform` classes just to modify date values. Since there is a clear distinction between snapshot (comes as string) and actions (comes as microseconds/int), this implementation should suffice.